test/unit/specs/composables/useBadgeData.spec.js

```diff
@@ -822,4 +822,192 @@ describe('useBadgeData.js', () => {
 			expect(getEarnedBadgeExplanation(ID_2BB, target)).toEqual('for helping us reach $2b in total impact');
 		});
 	});
+
+	describe('getFilteredLoansByJourney', () => {
+		it('should return expected filtered loans for us-economic-equality', () => {
+			const { getFilteredLoansByJourney } = useBadgeData();
+
+			const loans = [
+				{
+					id: 1,
+					geocode: {
+						country: {
+							isoCode: 'US'
+						}
+					}
+				},
+				{
+					id: 2,
+					geocode: {
+						country: {
+							isoCode: 'PR'
+						}
+					}
+				},
+				{
+					id: 3,
+					geocode: {
+						country: {
+							isoCode: 'PE'
+						}
+					}
+				},
+			];
+			expect(getFilteredLoansByJourney({ id: ID_US_ECONOMIC_EQUALITY }, loans)).toEqual([
+				{
+					id: 1,
+					geocode: {
+						country: {
+							isoCode: 'US'
+						}
+					}
+				},
+				{
+					id: 2,
+					geocode: {
+						country: {
+							isoCode: 'PR'
+						}
+					}
+				}
+			]);
+		});
+
+		it('should return expected filtered loans for climate-action', () => {
+			const { getFilteredLoansByJourney } = useBadgeData();
+
+			const loans = [
+				{
+					id: 1,
+					tags: [
+						'Test'
+					]
+				},
+				{
+					id: 2,
+					tags: [
+						'#Eco-friendly'
+					]
+				},
+				{
+					id: 3,
+					tags: [
+						'Agriculture'
+					]
+				},
+			];
+			expect(getFilteredLoansByJourney({ id: ID_CLIMATE_ACTION }, loans)).toEqual([
+				{
+					id: 2,
+					tags: [
+						'#Eco-friendly'
+					]
+				}
+			]);
+		});
+
+		it('should return expected filtered loans for womens-equality', () => {
+			const { getFilteredLoansByJourney } = useBadgeData();
+
+			const loans = [
+				{
+					id: 1,
+					gender: 'female'
+				}, {
+					id: 2,
+					gender: 'male'
+				}
+			];
+
+			expect(getFilteredLoansByJourney({ id: ID_WOMENS_EQUALITY }, loans)).toEqual([
+				{
+					id: 1,
+					gender: 'female'
+				}
+			]);
+		});
+
+		it('should return expected filtered loans for refugee-equality', () => {
+			const { getFilteredLoansByJourney } = useBadgeData();
+
+			const loans = [
+				{
+					id: 1,
+					themes: [
+						'Refugees/Displaced'
+					]
+				},
+				{
+					id: 2,
+					themes: [
+						'Education'
+					]
+				}
+			];
+
+			expect(getFilteredLoansByJourney({ id: ID_REFUGEE_EQUALITY }, loans)).toEqual([
+				{
+					id: 1,
+					themes: [
+						'Refugees/Displaced'
+					]
+				}
+			]);
+		});
+
+		it('should return expected filtered loans for basic-needs', () => {
+			const { getFilteredLoansByJourney } = useBadgeData();
+
+			const loans = [
```

Comment:
Add a loan here with theme as well, to cover this more unusual filter requirement when compared to the other milestones.